### PR TITLE
Edu VIC WA NT Starters: Version 1.000 added



### PR DESCRIPTION
**Pull request by simoncozens**
Taken from the upstream repo https://github.com/SorkinType/VICWANTSchoolHandAustralia at commit https://github.com/SorkinType/VICWANTSchoolHandAustralia/commit/abab94d19985f24132e45db4c4822b042eb63302.
## PR Checklist:

- [x] Family categorization tags collected from the type design team with the Categories Form
~- [ ] Minisite_url definition in the METADATA.pb file for commissioned projects~
~- [ ] Primary_script definition in the METADATA.pb file for all projects that have a primary non-Latin based language support target~
- [ ] Fontbakery checks are reviewed and failing checks are resolved in collaboration with the upstream font development team
- [ ] Diffenator2 regression checks for revisions on all projects that are currently in production
- [ ] Designers bio info have to be present in the designer catalog (at least an issue should be opened for tracking this, if they are not)
- [ ] Check designers order in metadata.pb, since the first one of the list appears as “principal designer”
- [ ] Social media formatted visual assets for all new commissioned projects in the Drive directory, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet
- [ ] Social media content draft for all new commissioned projects in the Drive directory and Social Media tracker spreadsheet, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet

### CODE REVIEW

**Comment by vv-monsalve** [comment]
@simoncozens 

> - [ ]  Minisite_url definition in the METADATA.pb file for commissioned projects
> - [ ] Primary_script definition in the METADATA.pb file for all projects that have a primary non-Latin based language support target

Being this a commissioned project, does it have a minisite URL? Please add it or cross out these points in the list for all the related PRs

> 🔥 **FAIL** Shapes languages in all GF glyphsets.
> 🔥 FAIL GF_Latin_Core glyphset:
> Shaper didn't attach acutecomb to j

The font should use soft-dotted glyphs to ensure on-the-fly composite glyphs.

**Comment by vv-monsalve** [comment]
Is this rendering ok?

Regular-diffbrowsers_text-Windows-10-10 0 20348-SP0_firefox_123 0 1
SemiBold-diffbrowsers_text-macOS-12 7 3-x86_64-i386-64bit_firefox_122 0 1

File: ofl/eduvicwantstarters/OFL.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Copyright 2023 The VIC WA NT School Hand Australia Project Authors (https://github.com/SorkinType/VICWANTSchoolHandAustralia)
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
+https://scripts.sil.org/OFL
```
Comment (vv-monsalve) [inline ofl/eduvicwantstarters/OFL.txt:5]: The url should be https://openfontlicense.org as it is in the font name table (IDs 13 and 14).
Don't know why FB is not reporting this mismatch.

**Review by vv-monsalve — DISMISSED** [review]

**Comment by simoncozens** [comment]
I'm not sure this is a commissioned project.

**Comment by vv-monsalve** [comment]
> I'm not sure this is a commissioned project.

I understood these were the Briem fonts, weren't they? I linked that issue in the development section of this PR. If it doesn't match the font, please modify that link to match the corresponding one.

**Comment by simoncozens** [comment]
This is not related to Briem; it's the Australia School fonts, an existing libre font project: https://www.australianschoolfonts.com.au/

**Comment by vv-monsalve** [comment]
Is there an open issue for these fonts? Could you please add it to the Development section of the PRs as well as the required labels (e.g. if they are new or upgraded fonts) on all the PRs. This is required at onboarding time to review them better.

File: ofl/eduvicwantstarters/OFL.txt (hunk 1 above)
Comment (vv-monsalve) [reply ofl/eduvicwantstarters/OFL.txt:5]: @m4rc1e suggested to make a PR to update all the OFLs in one shot. #7482

**Comment by vv-monsalve** [comment]
@EbenSorkin some outlines have the wrong direction and are creating for some shapes to not be visible (see the counter in the above image).

**Comment by vv-monsalve** [comment]
Given that special variants like "dotted", "arrows" or "starters" are applied only to A-Z a-z and 0-9, we'll check with @davelab6 about how to proceed with the fonts.

**Comment by vv-monsalve** [comment]
> we'll check with @davelab6 about how to proceed with the fonts.

Done in https://github.com/google/fonts/issues/7484#issuecomment-2052036141

**Pull request closed without merging**